client/extensions/hello-dolly/hello-dolly-page.js

```diff
@@ -0,0 +1,101 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+
+/**
+ * Internal Dependencies
+ */
+import Main from 'components/main';
+import Button from 'components/button';
+import Card from 'components/card';
+import SectionHeader from 'components/section-header';
+import FAQ from 'components/faq';
+import FAQItem from 'components/faq/faq-item';
+import { getCurrentLyric } from './state/selectors';
+import lyrics from './state/lyrics';
+import { nextLyric } from './state/actions';
+
+class HelloDollyPage extends Component {
```

Comment:
The whole point of this PR is to establish redux state in extension code, using hello-dolly as an example. So I'm not sure how you feel state doesn't belong in this component when it needs to both display the current lyric and also dispatch an action. I suppose we could pass it up the chain to the index, but does it really make sense there? The index doesn't care about lyrics. This component does.